styles/components/_navigation.scss

```diff
@@ -0,0 +1,145 @@
+@import "../abstracts/variables";
+@import "../base/base";
+
+.navegation {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  display: flex;
+
+  &__item {
+    margin-right: 22px;
+    text-decoration: none;
+
+    li {
+      color: var(--tuscany);
+      display: block;
+      padding: 4px;
+      z-index: 1;
+      font-size: 1rem;
+      text-decoration: none;
+      letter-spacing: 0.005em;
+      position: relative;
+      font-weight: normal;
+      transition: all 0.5s;
+    }
+
+    li::after {
+      display: block;
+      position: absolute;
+      top: 0;
+      left: 0;
+      bottom: 0;
+      right: 0;
+      margin: auto;
+      width: 100%;
+      height: 1px;
+      content: ".";
+      color: transparent;
+      background: var(--coral);
```

Comment:
I would suggest that if you are going to add colors to the background, is better to use background-color instead of background.